hal/lib/athena/ChipObject.h

```diff
@@ -13,12 +13,6 @@
 #include <stdint.h>
 
 #include "FRC_FPGA_ChipObject/RoboRIO_FRC_ChipObject_Aliases.h"
```

Comment:
Should the `NOLINT` be `NOLINT(build/include_order)`, or are other lint errors triggered by these?